### PR TITLE
Next

### PR DESCRIPTION
**Pull request by toniocodo**
THis PR brings
- Gnosis Safe support
- Rebasing opt-in options for OETH and OUSD
- Simplify migration by removing zapper contract
- Library updates
- Various style fixes

### CODE REVIEW

File: apps/defi/src/components/Topnav/components/Topnav.tsx [hunk 1]
```diff
@@ -34,10 +42,15 @@ export const Topnav = () => {
   const isSm = useMediaQuery(theme.breakpoints.down('md'));
   const accountMenuAnchorEl = useRef(null);
   const [accountMenuOpen, setAccountMenuOpen] = useState(false);
+  const alertMenuAnchorEl = useRef(null);
+  const [alertMenuOpen, setAlertMenuOpen] = useState(false);
   const [drawerOpen, setDrawerOpen] = useState(false);
   const { status, pendingCount } = useActivitiesStatus();
+  const isOethRebaseRequired = useIsRebaseBannerVisible(tokens.mainnet.OETH);
```
Comment (apexearth) [inline apps/defi/src/components/Topnav/components/Topnav.tsx:49]: maybe better var names
`isNonRebasingOETH`
`isNonRebasingOUSD`

A rebase is a single action so the name used here seems a little off.

File: libs/defi/shared/src/components/Activities/constants.tsx [hunk 2]
```diff
@@ -361,6 +362,35 @@ export const activityOptions: Record<ActivityType, ActivityOption> = {
       return <TransferIcon tokenIn={tokenIn} tokenOut={tokenOut} />;
     },
   },
+  rebasing: {
+    title: (activity, intl) =>
+      ({
+        pending: intl.formatMessage({ defaultMessage: 'Enabling rebasing' }),
+        signed: intl.formatMessage({ defaultMessage: 'Enabling rebasing' }),
+        success: intl.formatMessage({ defaultMessage: 'Rebase enabled' }),
```
Comment (apexearth) [inline libs/defi/shared/src/components/Activities/constants.tsx:370]: > Rebasing enabled

File: libs/defi/shared/src/components/Banners/RebasingBanner.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+          overflow: 'hidden',
+          backgroundColor: 'primary.faded',
+          ...rest?.sx,
+        }}
+      >
+        <TokenIcon token={token} sx={{ fontSize: 36 }} />
+        <Typography textAlign="center" noWrap={!isSmall}>
+          {intl.formatMessage({
+            defaultMessage:
+              'It looks like you are minting from a contract and have not opted into yield. You must opt-in to receive yield.',
```
Comment (apexearth) [inline libs/defi/shared/src/components/Banners/RebasingBanner.tsx:56]: > It looks like you are minting from a contract and have not opted into yield. Contracts must opt-in to receive yield.
maybe

Comment (toniocodo) [reply libs/defi/shared/src/components/Banners/RebasingBanner.tsx:56]: sounds better indeed!

**Pull request merged**